package/MDAnalysis/__init__.py

```diff
@@ -205,21 +205,7 @@
 
 from .due import due, Doi, BibTeX
 
-due.cite(BibTeX((
-            "@inproceedings{gowers2016, "
-            "title={MDAnalysis: A Python package for the rapid analysis "
-            "of molecular dynamics simulations}, "
-            "author={R. J. Gowers and M. Linke and "
-            "J. Barnoud and T. J. E. Reddy and M. N. Melo "
-            "and S. L. Seyler and D. L. Dotson and J. Domanski, and "
-            "S. Buchoux and I. M. Kenney and O. Beckstein},"
-            "journal={Proceedings of the 15th Python in Science Conference}, "
-            "pages={102-109}, "
-            "year={2016}, "
-            "editor={S. Benthall and S. Rostrup}, "
-            "note={Austin, TX, SciPy.} "
-            "}"
-            )),
+due.cite(Doi("10.25080/majora-629e541a-00e"),
```

Comment:
I tested that this gives a comparable citation (I think the editors were missing but otherwise seemed complete.)